src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java

```diff
@@ -17,8 +43,108 @@
         "uk.gov.hmcts.reform.professionalapi",
 })
 @SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
-public class Application  {
+public class Application  implements CommandLineRunner {
+
+    private static final String BLOB_URL = "http://3fbfcfb8181346cabaf7.blob.core.windows.net";
+    private static final String BLOB_HTTPS_URL = "https://3fbfcfb8181346cabaf7.blob.core.windows.net";
+    private static final boolean isEnableHttps = false;
+    private static final String ASURE_ACCOUNT_NAME = "3fbfcfb8181346cabaf7";
+    private static final String ASSURE_ACCOUNT_KEY = "pMfqbiimUan/k6q+G3t1rOpfBLSn2SeHV4AXZa+7XtNAKAdjKLbK7yl4/UU77NCU+taymTLN4eKyE/IIyTwolg==";
+    private static final String CONTAINER_NAME = "blobcontainer";
+
     public static void main(final String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    public void run(String... var1) throws IOException, InvalidKeyException {
+
+        Timer timer = new Timer();
+        timer.schedule(new TimerTask() {
+            public void run() {
+                try {
+                    pushFile();
+                } catch (IOException e) {
+                    log.error("IOException", e);
+                } catch (InvalidKeyException e) {
+                    log.error("InvalidKeyException", e);
+                }
+            }
+        }, 0, 60 * 1000 * 10);
+    }
+
+    public void pushFile() throws IOException, InvalidKeyException {
+
+        final String sourceCsvFile = "content.csv";
+        final File sourceFile = new File(this.getClass().getClassLoader().getResource(sourceCsvFile).getFile());
+        ServiceURL serviceUrl = createServiceUrl(new PipelineOptions());
+        ContainerURL containerUrl = serviceUrl.createContainerURL(CONTAINER_NAME);
+        createContainer(containerUrl);
+        final BlockBlobURL blockBlobUrl = containerUrl.createBlockBlobURL(sourceCsvFile);
+        uploadFile(blockBlobUrl, sourceFile);
+    }
+
+    @SuppressWarnings("deprecation")
+    public static void uploadFile(BlockBlobURL blob, File sourceFile) throws IOException {
+        log.info("Start uploading file %s...", sourceFile);
```

Comment:
why we are using deprectaed methods/code?

Reply:
Replaced with ignoreElements()